sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Release History
+## 1.0.1 (Unreleased)
```

Comment:
@JimSuplizio Any clue on why this was added?

Reply:
This was because @mssfang updated the changelog for the new version but didn't update the version in version_client.txt. @praveenkuttappan is looking into updating the script so that this wouldn't happen. For this particular file, I'm just going to edit the PR to remove the changes and it'll go away.